

class NetworkException extends AuthException {
  const NetworkException(super.message);
}